core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/impl/io/readJson.kt

```diff
@@ -229,18 +239,18 @@ internal fun fromJsonListAnyColumns(
 
                             v.longOrNull != null -> collector.add(v.long)
 
-                            v.doubleOrNull != null -> collector.add(v.double)
-
                             v.floatOrNull != null -> collector.add(v.float)
 
+                            v.doubleOrNull != null -> collector.add(v.double)
```

Comment:
@zaleslaw It's because of this change that we can now get floats out of json too instead of just doubles.